File: app/Models/Website.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;

class Website extends Model
{
    use HasFactory;

    // 白名单
    protected $fillable = ['name', 'status', 'path', 'php', 'ssl', 'ssl_date', 'note', 'created_at', 'updated_at'];
}

File: app/models/website.go
package models

import "github.com/goravel/framework/database/orm"

type Website struct {
	orm.Model
	Name   string `gorm:"not null;unique" json:"name"`
	Status bool   `gorm:"not null;default:true" json:"status"`
	Path   string `gorm:"not null" json:"path"`
	PHP    int    `gorm:"not null" json:"php"`
	SSL    bool   `gorm:"not null" json:"ssl"`
	Remark string `gorm:"not null" json:"remark"`

	Cert *Cert `gorm:"foreignKey:WebsiteID" json:"cert"`
}
